app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @posts= Post.find.(params(:id))
+  end
+
+  def new
+    @posts = Post.new
+  end
+  def create
+    # @posts = Post.find(params(:id))
```

Comment:
You'll need to use `.create` in order to make a post.

This will require setting up a `post_params` strong params method, as we've done in class. Let me know if you need any help with this.